tests/output/ConfWriterTest.py

```diff
@@ -35,16 +35,24 @@ def setUp(self):
         self.conf_parser = ConfParser()
         self.write_file_name = os.path.join(tempfile.gettempdir(),
                                             'ConfWriterTestFile')
+        self.write_dir_name = os.path.join(tempfile.gettempdir(),
+                                           '')
         self.uut = ConfWriter(self.write_file_name)
+        self.uut_dir = ConfWriter(self.write_dir_name)
 
     def tearDown(self):
         self.uut.close()
         os.remove(self.file)
         os.remove(self.write_file_name)
+        self.uut_dir.close()
+        os.remove(self.write_dir_name + '/.coafile')
 
     def test_exceptions(self):
         self.assertRaises(TypeError, self.uut.write_section, 5)
 
+    def test_exceptions_dir(self):
+        self.assertRaises(TypeError, self.uut_dir.write_section, 5)
```

Comment:
this test is unneeded since the exception has nothing to do with the dir, right?